darts/models/forecasting/tft_model.py

```diff
@@ -559,15 +560,22 @@ def __init__(self,
             Fraction of neurons afected by Dropout.
         hidden_continuous_size : int
             default for hidden size for processing continuous variables
-        add_cyclic_encoder : optional str
-            If other than None, apply cycling encoding to an attribute of the time index and add it to the
-            `future_covariates`.
-            This allows to use the TFTModel without having to pass future_covariates to `fit()` and `train()`
-            Must be an attribute of `pd.DatetimeIndex`, or `week` / `weekofyear` / `week_of_year` - e.g. "month",
-            "weekday", "day", "hour", "minute", "second". See all available attributes in
-            https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DatetimeIndex.html#pandas.DatetimeIndex.
-            For more information, check out :meth:`datetime_attribute_timeseries()
-            <darts.utils.timeseries_generation.datetime_attribute_timeseries>`
+        add_encoders : optional Dict
+            If other than None, add index encoding/s to the past and/or future covariates for training and prediction.
+            The dict must follow this convention:
+                `{encoder keyword: {temporal keyword: List[attributes]}}`
+            Supported encoder keywords:
+                `cyclic` for cyclic temporal encoder. See the docs :meth:`CyclicTemporalEncoder
+                <darts.utils.data.encoders.CyclicTemporalEncoder>`;
+                `datetime_attribute` for adding scalar information of pd.DatetimeIndex attribute. See the docs
```

Comment:
Question: is there a way to add the "naive" index values for integer-indexed series?

Reply:
Yes, there is

Reply:
How would this work?

Reply:
Implementing it right now :)